### DISCUSSION TITLE
Can I link your repo as a resource in upstream's README?

### DISCUSSION THREAD

**Discussion by aidandenlinger** [RelativeSure/autocopr#131] [General]
Hey again! Hope you're doing well :) I'd love to link your repo in `autocopr`'s README, as a resource for other devs to check out if they want to see the cool things you're doing with webhooks and Github Actions that I'm not exploring. Totally fine if you'd prefer that I didn't though!

**Comment by RelativeSure** [comment]
Hey @aidandenlinger,
Ofcourse go right ahead!

Sorry I havent responded for a few days, been sick and busy in life aha

Best regards,
RelativeSure
